README.md

```diff
@@ -59,7 +59,7 @@ After installing, once you restart Xcode, there will be a new option under the `
 RIGHT NOW!!!
 
 #Who
-Initially a [VOKAL](http://www.vokalinteractive.com) Hack Days Fall 2014 Project by [Ellen Shapiro](http://github.com/designatednerd). Massively improved by the desktop programming skillz of [Isaac](http://github.com/ilg) [Greenspan](http://github.com/vokal-isaac). 
+Initially a [Vokal](http://www.vokalinteractive.com) Hack Days Fall 2014 Project by [Ellen Shapiro](http://github.com/designatednerd). Massively improved by the desktop programming skillz of [Isaac](http://github.com/ilg) [Greenspan](http://github.com/vokal-isaac). 
```

Comment:
Note that the link has to remain vokalinteractive for now because vokal.io is still just showing the `pageok` thing.